_scripts/deploy.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+docker login -e "$DOCKER_EMAIL" -u "$DOCKER_USER" -p "$DOCKER_PASSWORD"
+REGISTRY="" make push
+docker login -e "$QUAY_EMAIL" -u "$QUAY_USER" -p "$QUAY_PASSWORD" quay.io
+make push
```

Comment:
For consistency with deis/docker-go-dev, this `deploy.sh` script probably belongs in `_scripts/ci/deploy.sh`